packages/rancher-pushprox/charts/templates/pushprox-servicemonitor.yaml

```diff
@@ -6,13 +6,7 @@ metadata:
   namespace: {{ template "pushprox.namespace" . }}
   labels: {{ include "pushProxy.serviceMonitor.labels" . | nindent 4 }}
 spec:
-  endpoints:
-  - port: metrics
-    proxyUrl: {{ template "pushProxy.proxyUrl" . }}
-    {{- if .Values.clients.https.enabled }}
-    params:
-      _scheme: [https]
-    {{- end }}
+  endpoints: {{include "pushProxy.serviceMonitor.endpoints" . | nindent 4 }}
```

Comment:
`[port: metrics]` is now the default values.yaml here, so we end up with the same exact code as before.